src/activity-log/index.html

```diff
@@ -17,13 +17,13 @@
     {% import 'macros/category-icon.html' as category_icon %}
     {% from 'post-macros.html' import pagination as pagination with context %}
 
-    <h1>Activity Log</h1>
-    <p class="h3">
+    <h1 data-qa-hook="title">Activity Log</h1>
```

Comment:
Does this necessarily need a qa hook or could the h1 be used? Or is this to avoid if an additional h1 is added to the page in the future? Perhaps make the hook name more specific. E.g. `main-title` (and `main-summary` and main-filter`).